impl/src/main/java/org/kuali/rice/kns/web/struts/action/KualiRequestProcessor.java

```diff
@@ -1,781 +1,787 @@
-		if ( LOG.isInfoEnabled() ) {
-			LOG.info(new StringBuffer("Started processing request: '").append(request.getRequestURI()).append("' w/ query string: '").append(request.getQueryString()).append("'"));
-		}
-
-		try { 
-			super.process(request, response);
-		} catch (FileUploadLimitExceededException e) {
-			ActionForward actionForward = processException(request, response, e, e.getActionForm(), e.getActionMapping());
-			processForwardConfig(request, response, actionForward);
-		} finally {
-			GlobalVariables.setKualiForm(null);
-		}
-
-		try {
-			ActionForm form = WebUtils.getKualiForm(request);
-			
-			if (form != null && form instanceof KualiDocumentFormBase) {
-				String docId = ((KualiDocumentFormBase) form).getDocId();
-				if (docId != null) { MDC.put(MDC_DOC_ID, docId); }
-			}
-
-			String refreshCaller = request.getParameter(KNSConstants.REFRESH_CALLER);
-			if (form!=null && KualiDocumentFormBase.class.isAssignableFrom(form.getClass()) 
-					&& !KNSConstants.QUESTION_REFRESH.equalsIgnoreCase(refreshCaller)) {
-				KualiDocumentFormBase docForm = (KualiDocumentFormBase) form;
-				Document document = docForm.getDocument();
-				String docFormKey = docForm.getFormKey();
-
-				UserSession userSession = (UserSession) request.getSession().getAttribute(KNSConstants.USER_SESSION_KEY);
-
-				if (WebUtils.isDocumentSession(document, docForm)) {
-					getSessionDocumentService().setDocumentForm(docForm, userSession, request.getRemoteAddr());
-				}
-
-				Boolean exitingDocument = (Boolean) request.getAttribute(KNSConstants.EXITING_DOCUMENT);
-
-				if (exitingDocument != null && exitingDocument.booleanValue()) {
-					// remove KualiDocumentFormBase object from session and
-					// table.
-					getSessionDocumentService().purgeDocumentForm(docForm.getDocument().getDocumentNumber(), docFormKey, userSession, request.getRemoteAddr());
-				}
-			}
-
-			if ( LOG.isInfoEnabled() ) {
-				LOG.info(new StringBuffer("Finished processing request: '").append(request.getRequestURI()).append("' w/ query string: '").append(request.getQueryString()).append("'"));
-			}
-
-		} finally {
-			// MDC docId key is set above, and also during super.process() in the call to processActionForm
-			MDC.remove(MDC_DOC_ID);
-			// MDC user key is set above, although it may have already been set.
-			if (MDC.get(MDC_USER_ALREADY_SET) == null) {
-				MDC.remove(MDC_USER);
-			} else {
-				MDC.remove(MDC_USER_ALREADY_SET);
-			}
-		}
-
-	}
-
-	/**
-	 * override of the pre process for all struts requests which will ensure
-	 * that we have the appropriate state for user sessions for all of our
-	 * requests, also populating the GlobalVariables class with our UserSession
-	 * for convenience to the non web layer based classes and implementations
-	 */
-	@Override
-	protected boolean processPreprocess(HttpServletRequest request, HttpServletResponse response) {
-		UserSession userSession = null;
-		if (!isUserSessionEstablished(request)) {

```

Comment:
getAuthenticatedPrincipalName was removed from the Identity Service.  The following line is the only line that changed in this class.